packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -29,6 +29,15 @@ const StaticQuery = props => (
   </StaticQueryContext.Consumer>
 )
 
+const useStaticQuery = query => {
+  const context = useContext(StaticQueryContext)
+  if (context[query] && context[query].data) {
+    return context[query].data
+  } else {
```

Comment:
nit: else not required

Reply:
Removed it but added it back now because 
- linter wants consistent returns
- generally more readable _imo_

Reply:
That should still be a consistent return though? Does ESLint really fail on something like this?

```js
const useStaticQuery = condition => {
  if (condition) {
    return { data: {} }
  }

  return null
}
```

That _is_ a consistent return! But - more a question, _not_ required for this PR. Keep it as is!

Reply:
Oh, gotcha! Done!